changelog-maker.js

```diff
@@ -51,6 +58,16 @@ function replace (s, m) {
   return s
 }
 
+function showUsage (removeMd) {
```

Comment:
yes, just help to give it sense and meaning. I can do that later on, i m not short of one or two more pr on that topic ahah :S